km_api/know_me/tests/models/test_image_content_model.py

```diff
@@ -8,11 +8,8 @@ def test_create_image_content(
     """
     Test creating image content for a profile item.
     """
-    image_gallery_item = media_resource_factory(file=image)
-
     models.ImageContent.objects.create(
         description='Test item description.',
-        image_resource=image_gallery_item,
```

Comment:
These gallery items were misnamed, but they shouldn't be deleted. We should change `image_gallery_item` to `image_resource` instead.